torch/csrc/jit/script/schema_matching.cpp

```diff
@@ -580,11 +577,16 @@ Value* emitBuiltinCall(
     throw error;
   }
 
-  throw ErrorReport(loc) << "Arguments for call are not valid.\n"
-                         << "The following operator variants are available:\n"
-                         << prefixLine(failure_messages.str(), "  ")
-                         << "\nThe original call is";
+  auto matched = matchSchemas(schemas, loc, graph, inputs, attributes, self);
+
+  if (matched.first < variants.size()) {
+    return emitBuiltinNode(matched.second, loc, graph, name);
+  } else {
+    Function* fn = builtin_functions[matched.first - variants.size()];
+    return insertGraph(graph, *fn->graph(), matched.second.inputs).at(0);
```

Comment:
keep comment around ? 
> we inline builtin calls because they are normally very small  // wrappers and are not useful for keeping around to debug

Reply:
Context: i ran into this when trying to add a builtin_functions with a loop that failed and the comment was helpful